src/init/init.ts

```diff
@@ -124,11 +124,7 @@ function createYeomanEnvironment(): Promise<any> {
     env.registerStub(ApplicationGenerator, 'polymer-init-application:app');
     env.registerStub(shopGenerator, 'polymer-init-shop:app');
     env.registerStub(pskGenerator, 'polymer-init-starter-kit:app');
-    env.lookup((error) => {
-      if (error) {
-        reject(error);
-        return;
-      }
+    env.lookup(() => {
```

Comment:
So I made this change because upgrading to typescript 2.0 exposed that this callback doesn't actually return an error in its type definition.

BUT looking again just now, and I realized that this was coming from our own custom type declaration and not an official one. The official documentation actually does mention that an error can be returned (although if you drill into the code even further, it's always null).

I'll revert this change and update the type declaration, thanks for catching this!